### ISSUE TITLE
Dataset and other files accessed in source code

### ISSUE THREAD

**Issue by ajeyabsf** [BalytskyiJaroslaw/RAPID-Net#2]
Hello. I am trying to run the RAPIDNet pipeline. While I am able to run the example run through files where the weights are downloaded from zenodo, I am unable to run the training pipeline related files, as they are referencing files not in the repo.
I have a few queries:
1. Which was the sc-PDB dataset you referenced? http://bioinfo-pharma.u-strasbg.fr/scPDB/ seems to be down always. https://drugdesign.unistra.fr/scPDB/ does have the dataset but the example file 1a27_1 you mentioned does not seem to be there.
2. Does the training pipeline have only 1 ensemble model run (in Soft_Dice_Relu_5020_Run_5)(Was the same cell run again with the number changed from 1 to 5)? And is it part of PUResnet or Rapidnet ( as it is saved in a folder named "PUResnet")?
3. Is there a way to get the npy files referenced in training?

**Comment by BalytskyiJaroslaw** [comment]
Hello! 

1) For training, we used a subset of scPDB, you can download it here: 
https://github.com/jivankandel/PUResNet/blob/main/scpdb_subset.zip

2) Trained weights for all 5 models are here: https://zenodo.org/records/14796981

3) As a crosscheck, the data shape should be this: 

Input data shape: (5020, 36, 36, 36, 18)
Output data shape: (5020, 36, 36, 36, 1)

Also, as a crosscheck, class weights should be these: 

Class weights: {0: 0.1671444675365879, 1: 114.95844646927513, 2: 156.0054192527346, 3: 544.9148472834887, 4: 4819.2, 5: 5576502.857142857}

BTW, is "Open in Colab" link working? See the screenshot attached. 

<img width="1195" height="401" alt="Image" src="https://github.com/user-attachments/assets/6bbbc498-3b11-482e-987e-9a8fede95720" />

When you press it, 

First, run the cell :🔧 Set Up the Environment in Colab (Open Babel) ||=> This install openbabel. 

Then, run the cell: 🦁 Initialize the RAPID-Net Model

After this: 🎉 RAPID-Net model 🦁 unleashed and ready for pocket prediction! 🚀

Enter the Job_name (the name of your protein). 

There is an example for ABHD5, but you can plug your favorite one. 

Does it work?

**Comment by BalytskyiJaroslaw** [comment]
I will upload the processed training data and labels to Zenodo tomorrow, it is approximately 16 GB.

**Comment by ajeyabsf** [comment]
Yes. The colab notebook runs fine with the weights being loaded. I was trying to go through the entire flow in the source code and was unable to connect the training data parts.

Thank you for uploading the training data and labels. Request you to let me know the link in zenodo when you upload.

A couple of queries:
- 1a27_1 is not in the subset data you have used too.
- Combined_UNCUT_preprocessing in Training script and Fixed_Preprocessing in preprocessing script point to the same thing right?

**Comment by BalytskyiJaroslaw** [comment]
Yep, will upload all the training data today, so you can crosscheck. 

Just in case, you can generate the training data and labels yourself using this notebook: 

https://github.com/BalytskyiJaroslaw/RAPID-Net/blob/main/training_pipeline/Training_Pockets_Data_Preprocessing.ipynb

This part: 

<img width="1173" height="656" alt="Image" src="https://github.com/user-attachments/assets/77468b1c-4f20-4c24-a727-0590ca1f2220" />

**Comment by BalytskyiJaroslaw** [comment]
Combined_UNCUT_preprocessing in Training script and Fixed_Preprocessing in preprocessing script point to the same thing right?

||=> 

Yes. That's just the folder where you store your training data and labels. Call it any way you like.

**Comment by ajeyabsf** [comment]
Got it. Thank you.

**Comment by BalytskyiJaroslaw** [comment]
Actually, 1a27_1 is present in the training set. See the txt files attached. 

[scPDB_orig_content.txt](https://github.com/user-attachments/files/23546464/scPDB_orig_content.txt)
[scpdb_subset_contents.txt](https://github.com/user-attachments/files/23546463/scpdb_subset_contents.txt)

The original training set is 17595 proteins. Filtered one is 5020 proteins.

**Comment by ajeyabsf** [comment]
Apologies. Missed it.

**Comment by ajeyabsf** [comment]
Apologies. I missed it

**Comment by ajeyabsf** [comment]
I will close the issue. Thank you for the information.
